packages/isolated-editor/client/components/block-editor-toolbar/more-menu/editor-menu.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuGroup, MenuItem } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { check } from '@wordpress/icons';
+
+/** @typedef {import('./index').OnClose} OnClose */
+/** @typedef {import('../../../index').BlockEditorSettings} BlockEditorSettings */
+
+/**
+ * Close dropdown callback
+ * @callback OnSetMode
+ * @param {string} mode
+ */
+
+/**
+ * The 'more menu' editor section
+ *
+ * @param {object} props - Component props
+ * @param {boolean} props.isCodeEditingEnabled
+ * @param {string} props.editorMode
+ * @param {OnClose} props.onClose - Callback to close the menu
+ * @param {OnSetMode} props.onSetMode
+ * @param {BlockEditorSettings} props.settings - Settings
+ */
+function EditorMenu( { onClose, editorMode, onSetMode, isCodeEditingEnabled, settings } ) {
+	/**
+	 * @param {string} mode
+	 */
+	const setMode = ( mode ) => {
+		onSetMode( mode );
+		onClose();
+	};
+
+	if ( ! isCodeEditingEnabled || ! settings.iso.moreMenu.editor ) {
+		return null;
+	}
+
+	return (
+		<MenuGroup label={ _x( 'Editor', 'noun' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( 'Editor', { context: 'Noun: A user role displayed in a badge'} )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/isolated-editor-reader-comments)


:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `Noun: A user role displayed in a badge`
- `User role`

 Would it make sense to reuse one of the above?